### PR TITLE
(refactor) use of the multi-catch construct

### PR DESCRIPTION
**Pull request by rbonifacio**

### CODE REVIEW

File: src/org/apache/commons/httpclient/HttpMethodDirector.java [hunk 1]
```diff
@@ -507,14 +507,7 @@ private void executeWithRetry(final HttpMethod method)
...
             }
-        } catch (IOException e) {
-            if (this.conn.isOpen()) {
-                LOG.debug("Closing the connection.");
-                this.conn.close();
-            }
-            releaseConnection = true;
-            throw e;
-        } catch (RuntimeException e) {
+        } catch(IOException | RuntimeException e) {
```
Comment (thc202) [inline src/org/apache/commons/httpclient/HttpMethodDirector.java]: Please, revert this change, this is a forked class, better keep the changes to a minimum and essential ones.

File: src/org/zaproxy/zap/extension/autoupdate/ExtensionAutoUpdate.java [hunk 2]
```diff
@@ -973,15 +973,11 @@ public void run() {
...
+				    		} catch(SSLHandshakeException | InvalidCfuUrlException e2) {
 					    		if (callback != null) {
 					    			callback.insecureUrl(longUrl, e2);
 					    		}
-							} catch (InvalidCfuUrlException e2) {
-					    		if (callback != null) {
-					    			callback.insecureUrl(longUrl, e2);
-					    		}
-							} catch (Exception e2) {
+							} catch(Exception e2) {
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/autoupdate/ExtensionAutoUpdate.java]: This indentation change could be reverted.

Comment (rbonifacio) [inline src/org/zaproxy/zap/extension/autoupdate/ExtensionAutoUpdate.java]: Dear @thc202, I will proceed according to your comments and (as soon as possible) I will update this pull request. Thanks!

**Comment by rbonifacio** [comment]
closing this pull request in order to (a) fix it according to the above comments and (b) send a new pull request.

**Comment by kingthorin** [comment]
@rbonifacio for future reference you can do an interactive rebase and fix-up or squash into a single  commit/PR.

- https://www.instructables.com/id/How-to-Rebase-Interactive-With-Eclipse-eGit/
- https://www.kernel.org/pub/software/scm/git/docs/git-rebase.html#_interactive_mode
- https://wiki.eclipse.org/EGit/User_Guide#Interactive_Rebase
